src/drive/web/modules/views/Public/index.jsx

```diff
@@ -170,6 +173,7 @@ const PublicFolderView = ({
           hasWriteAccess={hasWritePermissions}
           refreshFolderContent={refreshFolderContent}
         />
+        <SharingBanner />
```

Comment:
In https://github.com/cozy/cozy-drive/pull/2285/commits/e7f158a55e946efda8a69c386d9d7cd4c4b41db1#diff-92ce258d37c416c570baf4a15e40c8b72f0ed1b8535cf96b3b787c58f0f4bba6R12-R13 we have the "SharingBanner" above PublicToolbar. Here it's the contrary. Is it intended?

Reply:
Ce n'est plus le cas 1 commit plus tard